### PR TITLE
front: editor - tooltip and loader on map popup while select

### LINKED ISSUE #5591: Infra editor usability enhancement: tooltip and loader display
Here is the current experience when hovering the infrastructure on the map:
https://github.com/osrd-project/osrd/assets/870509/d6706141-932a-43aa-9a98-dd9fe1712d81

This display stuttering is happening because :
- the tooltip toggles between two sizes: one with the loader and one with infrastructure informations,
- the loader appears and disappears quickly because the informations are available… quickly.

The loader is a visual indication for "if you thought I was idle I'm not, and I'm currently working". When data can be displayed under a second, it's not necessary.

The tooltip should have one fixed size, and the loader should only be displayed after 700ms (this is a ballpark value we can fine-tune together) if the infra data isn't available.

### PR DESCRIPTION
**Pull request by sim51**
Close #5591

### CODE REVIEW

File: front/src/applications/editor/tools/selection/styles.scss [hunk 1]
```diff
@@ -3,3 +3,10 @@
     font-weight: bold;
   }
 }
+
+.popup.editor-selection {
+  .maplibregl-popup-content {
+    min-width: 200px;
+    min-height: 100px;
+  }
+}
```
Comment (clarani) [inline front/src/applications/editor/tools/selection/styles.scss:12]: missing new line

**Review by clarani — COMMENTED** [review]
Left some comments

**Review by clarani — CHANGES_REQUESTED** [review]
The first commit needs to be removed

**Comment by sim51** [comment]
See PR #5787

**Pull request closed without merging**